views/partials/footer.ejs

```diff
@@ -35,6 +41,8 @@
             <li class="ms-3"><a class="text-muted" href="https://www.instagram.com/hacettepeoyt/"><svg class="bi" width="24" height="24"><use xlink:href="#instagram"/></svg></a></li>
             <!--<li class="ms-3"><a class="text-muted" href=""><svg class="bi" width="24" height="24"><use xlink:href="#facebook"/></svg></a></li>-->
             <li class="ms-3"><a class="text-muted" href="https://github.com/hacettepeoyt"><svg class="bi" width="24" height="24"><use xlink:href="#github"></use></svg></a></li>
+            <li class="ms-3"><a class="text-muted" href="https://mastodon.social/@hacettepeoyt"><svg class="bi" width="24" height="24"><use xlink:href="#mastodon"></use></svg></a></li>
+            <li class="ms-3"><a class="text-muted" href="https://youtube.com/@hcettepeoyt"><svg class="bi" width="24" height="24"><use xlink:href="#youtube"></use></svg></a></li>
```

Comment:
There is a quick typo in the url. It should be like this:
"https://youtube.com/@hacettepeoyt" instead of "https://youtube.com/@hcettepeoyt"

Reply:
Just updated the link now :)